app/api/sessions.py

```diff
@@ -104,12 +106,24 @@ def validate_date(self, data, original_data):
         type_='speaker')
 
 
-class SessionList(ResourceList):
+class SessionListPost(ResourceList):
     """
-    List and create Sessions
+    List Sessions
     """
     def before_post(self, args, kwargs, data):
         require_relationship(['event'], data)
+        if not has_access('is_coorganizer', event_id=data['event']):
```

Comment:
An unauthenticated user can:
- View all accepted/approved sessions (even via relationships)

An authenticated non-event org user can:
- Create a session for any live event
- Session can have a state of either draft or pending. (Default pending)
- View/edit his/her own sessions (Cannot add/modify track/microlocation relation)
- Delete his/her own sessions

An event org can:
- Create a session with any state for that event
- List all sessions of that event
- Modify all sessions of that event and all relationships
- Delete all sessions

> event org referes to event organizer/co-organizer